### PR TITLE
add guide for 'upgrade by installing again' to close #18

### LINKED ISSUE #18: How to best update/upgrade to newest version of Qiime 2
Edit:

## The easiest way to update is to just install the newest version in a new conda environment by following the normal install instructions. 

The install instructions are [in the documentation](https://docs.qiime2.org).  

---

Original post: 

We used to have [official docs](https://docs.qiime2.org/2017.7/upgrade/#updating-qiime-2-core-2017-7) for updating / upgrading to the newest version of Qiime.

This could be helpful on the [forums](https://forum.qiime2.org/t/updating-qiime2-2018-2-to-2018-4/4017), and could be a simple link on the Qiime 2 conda [install page](https://docs.qiime2.org/2018.4/install/).

### PR DESCRIPTION
**Pull request by colinbrislawn**

### CODE REVIEW

File: 2018.6/_sources/install/native.txt [hunk 1]
```diff
@@ -87,3 +87,10 @@ Next steps
 Now that you have the Core distribution installed, check out the :doc:`q2cli docs <../interfaces/q2cli>` to get familiar with the QIIME 2 command-line interface (it is used extensively in the :doc:`tutorials <../tutorials/index>`). After that, try out the :doc:`QIIME 2 tutorials <../tutorials/index>` for examples of using QIIME 2 to analyze microbiome datasets. You might also try installing other QIIME 2 :doc:`interfaces <../interfaces/index>`.
 
 .. _`Miniconda`: https://conda.io/miniconda.html
+
+
+How do I update to the newest version of Qiime?
```
Comment (ebolyen) [inline 2018.6/_sources/install/native.txt]: `Qiime` -> `QIIME 2`

File: 2018.6/_sources/install/native.txt [hunk 2]
```diff
@@ -87,3 +87,10 @@ Next steps
...
+------------------------------------------------
+
+In order to to update/upgrade to the newest release, you just install the newest version in a new conda environment by following the instructions above. Then you will have two conda environments, one with the older version of Qiime and one with the newer version.
```
Comment (ebolyen) [inline 2018.6/_sources/install/native.txt]: `Qiime` -> `QIIME 2`

**Review by ebolyen — CHANGES_REQUESTED** [review]
Thanks @colinbrislawn! Just a term swap, otherwise LGTM.

**Comment by ebolyen** [comment]
Sorry to be nitpicky, but can QIIME be spelled in all caps? Thanks!

**Comment by colinbrislawn** [comment]
Updated! (Good catch. Clearly I didn't read the style guide...)

**Comment by thermokarst** [comment]
Oops, this PR is on the built docs, not the docs source! We should be making these changes at https://github.com/qiime2/docs - @colinbrislawn would you mind re-opening there? The same edits should go into https://github.com/qiime2/docs/blob/master/source/install/native.rst

Thanks!

**Comment by thermokarst** [comment]
Sorry, didn't mean to close, hit the wrong button. I would like to close, but only after you confirm @colinbrislawn!

**Comment by colinbrislawn** [comment]
Yes, I'll open a new PR on the source.

... Does this repo just holds the rendered documents, while all the source is somewhere else? Can we put a note of that somewhere?

**Comment by thermokarst** [comment]
Thanks @colinbrislawn!

> ... Does this repo just holds the rendered documents, while all the source is somewhere else? Can we put a note of that somewhere?

screen shot 2018-08-29 at 8 53 45 am

**Comment by colinbrislawn** [comment]
Oh that's exactly the warning I was suggesting.

I can't believe I missed that... 🤕

**Comment by ebolyen** [comment]
oh gosh... I completely missed this as well >_<

**Pull request closed without merging**